### PR TITLE
DEFINE ALL THE VOXELS!!

### PR DESCRIPTION
**Pull request by tomsons26**

### CODE REVIEW

File: sequences/vehicles.yaml [hunk 1]
```diff
@@ -18,10 +37,158 @@ cmin:
...
+	icon: xxicon
+		Start: 0
+
+sapc:
+	icon: sapcicon
+		Start: 0
+
+dtruck:
+	icon: trkaicon
+	Start: 0
```
Comment (abcdefg30) [inline sequences/vehicles.yaml]: Wrong indentation.

File: sequences/voxels.yaml [hunk 2]
```diff
@@ -1,183 +1,222 @@
...
 
-ttnk:
+dest:
 	idle:
+#Needs without weapon image destwo
 
-gatick:
-	turret: ttnktur
-	barrel: ttnkbarl
+#dmisl:	#Dreadnought Missile needs voxel projectiles
```
Comment (abcdefg30) [inline sequences/voxels.yaml]: `#dmisl: #Dreadnought Missile needs voxel projectiles` please.

File: sequences/voxels.yaml [hunk 3]
```diff
@@ -1,183 +1,222 @@
...
+
+naflak:
+	turret: flaktur	
+
+flata: #Unused
 	idle:
-	turret: hvrtur
 
-stnk:
+fv:	#Has different turrets per unit thats inside,default is patriot
```
Comment (abcdefg30) [inline sequences/voxels.yaml]: Same here (notab between : and # please).

File: sequences/voxels.yaml [hunk 4]
```diff
@@ -1,183 +1,222 @@
...
-#m113: # TODO: unused
-#	idle:
-#	turret: m113tur
+tnkd:
+	idle:
 
-#mlrs: # TODO: unused
-#	idle:
-#	turret: mlrstur
+tractor:#Unused
```
Comment (abcdefg30) [inline sequences/voxels.yaml]: Space between : and # this time, please.

File: sequences/voxels.yaml [hunk 5]
```diff
@@ -1,183 +1,222 @@
...
-#	turret: 1tnktur
-#	barrel: 1tnkbarl
+dtruck:
+	idle: trucka
 
-#2tnk: # TODO: unused
-#	idle:
-#	turret: 2tnktur
-#	barrel: 2tnkbarl
+truckb:#Psst Has crates in it :D
```
Comment (abcdefg30) [inline sequences/voxels.yaml]: Same here.
Btw, this comment seems unneccesary.

**Comment by Phrohdoh** [comment]
Looks good. Comment styles need to be fixed.
Haven't tested this yet though.

**Comment by abcdefg30** [comment]
Looks good to me now.

**Comment by abcdefg30** [comment]
Thanks! :)

**Pull request merged**